worker/codalabworker/docker_client.py

```diff
@@ -248,8 +248,13 @@ def remove_image(self, repo_digest):
                 for action, target in line.items():
                     logger.debug('docker image %s: %s', action.lower(), target)
 
+    ''' Download the specified docker image with tag/digest. If no tag is specified, downloads the latest '''
     @wrap_exception('Unable to download Docker image')
     def download_image(self, docker_image, loop_callback):
+        if len(docker_image.split(":")) < 2:
+            logger.debug('Missing tag/digest on request docker image "%s", defaulting to latest', docker_image)
+            docker_image = ":".join([docker_image, "latest"])
```

Comment:
nit: I think we are using single quotes everywhere

Reply:
fix'd